gleam.toml

```diff
@@ -6,8 +6,8 @@ repository = { type = "github", user = "gleam-community", repo = "colour" }
 gleam = ">= 0.33.0"
 
 [dependencies]
-gleam_stdlib = "~> 0.34"
+gleam_stdlib = ">= 0.50.0 and < 2.0.0"
 gleam_json = ">= 1.0.1 and < 3.0.0"
```

Comment:
This constraint would need bumping now because `json.parse` was only introduced in 2.2.0

Reply:
Thanks, version has been updated